mitodl/ocw-www
# Updates to search page design

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #64 

#### What's this PR do?
Updates styling according to issue

#### How should this be manually tested?
Go to `/search` and select some facets. It should look like the mockup in the linked issue

#### Screenshots (if appropriate)
(ignore the HTML images, those are just placeholders)

![Screenshot_2021-03-22 Screenshot](https://user-images.githubusercontent.com/863262/112058287-c5505780-8b30-11eb-9fad-53c8df48c6de.png)
